packages/core/src/index.ts

```diff
@@ -29,14 +36,16 @@ const ERROR = "error";
 const noOp = () => {};
 
 // will cover unhandled errors, js crashes
-const handleError = async (error: Error, callback: () => void) => {
+const handleError = async (error: Error | UIError, callback: () => void) => {
```

Comment:
shouldn't be needed I don't think, the new type can be handled inside `logIfComponentError`